include/executor/executor.h

```diff
@@ -124,7 +125,7 @@ class HostFuncHandler {
 
 /// Executor flow control class.
 class Executor {
-public:
+public: /// add another argument to this for coredump to initialise it.
```

Comment:
Why do we need this comment here? Is it a TODO item?

Reply:
Yes, it is a TODO item. Will label it accordingly.